### ISSUE TITLE
Why is the Prediction results all Nan?

### ISSUE THREAD

**Issue by hydxqing** [Never-less/SegAN#8]
After training, I saved the model parameters of NetS, and then the segmentation prediction is carried out with NetS model. Why is the result all Nan?

**Comment by YuanXue1993** [comment]
Maybe try with different learning rate? Are you using your own data?

**Comment by hydxqing** [comment]
Yes, I use my own data. Loss during training has generally been very small and shows large and small.What's the matter?

**Comment by YuanXue1993** [comment]
What do you mean by "shows large and small"? I suggest you start w/o the adversarial loss (e.g., use the generator with cross-entropy loss) to see if there are any bugs in the code. After running smoothly, you can move to the adversarial training.
